contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,382 @@
+---
+title: Identify
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+
+---
+
+Identify enables you to to track your users across different platforms, and connect events from before and after they log in for the first time.
```

Comment:
JavaScript (client) `identify` works a lot differently than server side `identify`. This definition works with the client side, not really the server side. [See this](https://posthog.com/tutorials/identifying-users-guide).